### DISCUSSION TITLE
What have / are you learning in November

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#462] [Poll] [answered]
Put your topics below and vote on other peoples 🤓

**Comment by iason9** [comment]
Algorithms

**Reply by dheerajdlalwani** [reply]
Ahh, JavaFX & JDBC for college projects and submissions. Really hectic week.

**Reply by Akshu-on-github** [reply]
Yup, my November was filled with algorithms and data structures

**Comment by BOLT04** [comment]
DevOps and creating CI/CD pipelines

**Comment by Agrover112** [comment]
Crypto Theory !

**Reply by BOLT04** [reply]
Not sure if this helps, but I think it's an amazing resource related to that area: https://www.youtube.com/watch?v=Dgj_OStjD1Q

**Comment by iason9** [comment]
TypeScript

**Comment by AllanRegush** [comment]
Computer Architecture

**Reply by BOLT04** [reply]
Awesome! Not sure if this is related, but I found this repo about System Design: https://github.com/donnemartin/system-design-primer.

If you mean more like hardware architecture, and Intel stuff... then that link won't help 😅

**Reply by AllanRegush** [reply]
Yeah, hardware.

**Comment by Vyvy-vi** [comment]
Flask

**Comment by iason9** [comment]
React.js

**Comment by alrifay** [comment]
[Vue.js](https://vuejs.org/)

**Comment by GreyhoundProject** [comment]
Github (I'm new to this)

**Comment by SKULLXL** [comment]
JavaScript Basics and using GitHub and Git and other tools.

**Comment by mikeysan** [comment]
Python and Postman (REST API)

**Comment by Akshu-on-github** [comment]
Well, apart from the things I usually do? **Assembly language**
It's surprisingly neat!

**Comment by eddiejaoude** [comment, marked as answer]
I think it is time for December 🤓

**Reply by mikeysan** [reply]
I agree

**Reply by eddiejaoude** [reply]
December created https://github.com/EddieJaoudeCommunity/support/discussions/510
